front-end-dep-tree/README.md

```diff
@@ -0,0 +1,99 @@
+Thank you for your interest in working with ActiveState as a developer. This
+programming assignment will help us to evaluate your skills relative to our
+working environment and is used as a primary tool in our hiring process. Later
+on in the hiring process we will have one of our developers pair with you on
+making changes to your code, fixing bugs, etc.
+
+We expect this assignment to occupy no more than 3 hours of your time.  We are
+primarily looking for high-quality code, including test code, and we recognize
+that you may not have time to fully polish your work, so focus on code before
+documentation and other niceties.
+
+We strongly encourage the use of third party packages as you work on this
+assignment. However, see the note at the end about providing us with
+instructions on installing/running your assignment.
+
+We are happy to accept solutions provided in the following languages:
+
+* Elm
```

Comment:
I'd change this to not imply there were other alternative.  Just say "We would prefer the assignment is done in Elm."

Then there is the note about doing it in a different language below as long as they ask permission.

Reply:
Fixed